### DISCUSSION TITLE
updating feature version in category.xml

### DISCUSSION THREAD

**Discussion by luvs2pr0gram** [eclipse-tycho/tycho#1267] [Q&A]
Greetings,

I have three plugin projects and their three feature projects with one update site.  They all have different versions ie 1.3.5, 2.7.5, etc.

Is there a way to sync the feature versions in update site's category.xml with the plugin/feature versions using maven/tycho?

**Comment by laeubi** [comment]
As far as I know you can simply omit the version and it will be synchronized on the build.

**Reply by luvs2pr0gram** [reply]
It synchronizes the qualifier time stamp but not the major/minor versions. I'll do it using some scripting. Thanks for the reply.

**Reply by laeubi** [reply]
I mean you can completely omit the version, that is provide no version at all.

**Reply by luvs2pr0gram** [reply]
Sorry, which version are you referring to? The Bundle-Version? Or the version in the POM?

**Reply by laeubi** [reply]
The version in the `category.xml`

**Reply by luvs2pr0gram** [reply]
Awesome, thanks!!

**Comment by mickaelistria** [comment]
Have you tried considered directly including the bundles in your category.xml so you could discard the features?

**Reply by luvs2pr0gram** [reply]
Thanks for the tip. I tried it but when I go to install using Install > New Software they don't show up on the list but instead get an error that the file could not be found.
